marlowe-playground-server/src/Lambda.hs

```diff
@@ -5,7 +5,10 @@ module Lambda where
 import           Aws.Lambda
```

Comment:
Do we actually want to keep the `Lambda.hs` files?

Reply:
that was my question, I think no. I think we will use bitte if we have any problems that lambda could solve. I'll rip it out